src/test/java/uk/gov/hmcts/ccd/WireMockBaseTest.java

```diff
@@ -62,6 +65,25 @@ public void verifyWireMock(int count, RequestPatternBuilder postRequestedFor) {
         wireMockServer.verify(count, postRequestedFor);
     }
 
+    public ObjectMapper objectMapper() {
```

Comment:
why can't we inject proper from application code `JacksonObjectMapperConfig`?

Reply:
I would argue that's just using the production code to test...

Reply:
agree... make sense now.